org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/ProcessDialog.java

```diff
@@ -312,10 +312,11 @@ private void hideBusyMask()
 	{
 		if (mask != null && mask.getParent() != null) {
 			mask.detach();
-			StringBuilder script = new StringBuilder("(function(){let w=zk.Widget.$('#");
-			script.append(getParent().getUuid()).append("');w.busy=false;");
-			script.append("})()");
-			Clients.response(new AuScript(script.toString()));
+			if(getParent() != null) {
+				StringBuilder script = new StringBuilder("var w=zk.Widget.$('#");
```

Comment:
WARNING: this is reverting the change from:
https://github.com/idempiere/idempiere/commit/895923f7f229e3c3195c691afdba3c12e6fe61ca#diff-45852062765555bb780d1af8808a6582852fa6a3102af104344b02c5b2bad61c